docs/pages/Theming/Tokens/Basics.md

```diff
@@ -31,13 +35,96 @@ const colorTableFluent: { [P in PersonaCoinFluentColor]: string } = {
 };
 ```
 
-\* An exception is that brand color definitions change depending on platform.
+### Font
+
+You can find tokens related to fonts such as weight and size in the `globalTokens.font` property.
+
+An example of usage is in our Badge component, where we specify different font sizes for different Badge sizes.
+
```

Comment:
I am not sure if this is how we want the font tokens to be accessed by partners, or would the recommendation be to use the typography tokens like caption1, caption2 (using theme ) to style controls if needed..? This way of accessing globals would not reflect any changes done by theme customisation, whereas the theme tokens access would - so I was recommending the Android crew to use the tokens from theme typography tokens like     `fontSize: t.typography.sizes.body,`. Most designs also do not directly use the font tokens from global tokens, instead use from alias tokens for styling controls.
I am wondering if there is a single recommendation on this? Badge usage for tokens this way is an outlier, or recommended? If it's an outlier, this section for tokens could better fit under section of alias tokens below?

Reply:
This query is also valid for any other global tokens which have alias layer - do we want to recommend usage directly from global tokens, or alias? I think it as later to support custom themes, but still learning :)

Reply:
I added this just based off how I know stroke width/corner radius tokens get used, but sounds like font/typography is a little more complicated. Thinking I'll remove this section altogether and create a work item for @amgleitman to add documentation for font/typography

Reply:
Removed the documentation I had added on font; made an issue on ADO assigned to @amgleitman

Reply:
Also to address whether we want to generally recommend usage directly from global tokens vs. alias, there was this line in the documentation already:

"In most cases you will not be referring to global tokens directly. The case where you may be referring to global tokens is when you're defining your own alias token, or need to access brand colors."

But I'll also bring this up offline to see what others think

Reply:
Was resolved offline - I've reorganized the page so that the different types of tokens are no longer grouped in either global or alias tokens, and for each type of token we can provide whatever recommended usage we want.